### ISSUE TITLE
Output nitpicks for alr, alr help, and alr --help

### ISSUE THREAD

**Issue by onox** [alire-project/alire#644]
Just a bunch of nitpicks:

`alr` shows:

```

Ada Library Repository manager
USAGE: alr [global options] command [command options] [arguments]


Valid commands:
```

* Note the empty line at the beginning (there's also an empty line at the end of the output) and usage is written in uppercase. There are 2 empty lines instead of 1 in the output.

While `alr --help` shows:

```
 
Usage: alr [global options] <command> [command options] [arguments]
```

* Empty line at the beginning again, but no empty line at the end. Usage is not fully uppercase (better).

Running `alr help` shows:

```
error: Please specific a single help keyword:

Valid commands:
```

* Error is printed instead of usage instruction.

This could be changed to (suggestion):

```
Usage: alr help <command|topic>

Valid commands:
```

Alr version: 0.8.0-dev
OS: Arch Linux

**Comment by onox** [comment]
What do you think of something like this? (output of `alr`)

![20210201_12h48m24s_grim](https://user-images.githubusercontent.com/3361585/106455869-1034e380-648d-11eb-9c3a-374cf06233f0.png)

**Comment by mosteo** [comment]
Looks great! It also makes me realize that there's a mix of imperatives/indicatives that rubs me the wrong way...

**Comment by onox** [comment]
Okay, I'll tune the output of `alr help` and `alr help <command>|<topic>` as well then.

**Comment by onox** [comment]
`alr` now looks like this: (added global options)

![20210206_01h57m28s_grim](https://user-images.githubusercontent.com/3361585/107103798-378a0880-681f-11eb-9d0c-8db7b867ef42.png)

**Comment by mgrojo** [comment]
Note that most Linux commands (GNU style guide?) use this format for short/long options:
```
Pattern selection and interpretation:
  -E, --extended-regexp     PATTERN is an extended regular expression
  -F, --fixed-strings       PATTERN is a set of newline-separated strings
  -G, --basic-regexp        PATTERN is a basic regular expression (default)
  -P, --perl-regexp         PATTERN is a Perl regular expression
  -e, --regexp=PATTERN      use PATTERN for matching
  -f, --file=FILE           obtain PATTERN from FILE
  -i, --ignore-case         ignore case distinctions
  -w, --word-regexp         force PATTERN to match only whole words
  -x, --line-regexp         force PATTERN to match only whole lines
  -z, --null-data           a data line ends in 0 byte, not newline
```

So it might be a good idea to maintain that format. The rest is clearly a perfect enhancement.

**Comment by mgrojo** [comment]
> Looks great! It also makes me realize that there's a mix of imperatives/indicatives that rubs me the wrong way...

After a quick review, imperative seems the norm.

**Comment by onox** [comment]
> Note that most Linux commands (GNU style guide?) use this format for short/long options

Are you referring to the comma? The formatting of the output is inspired by [Poetry](https://python-poetry.org/), a popular package manager for Python.

**Comment by mgrojo** [comment]
Yes, I was referring to the comma. It seems pretty standard.

**Comment by onox** [comment]
Merged in #688
